### DISCUSSION TITLE
Bulk / batch feature

### DISCUSSION THREAD

**Discussion by natole13** [Sanster/IOPaint#540] [Q&A]
I have thousands of watermarked images. A small, ridicule watermark in the top right. 

Does IOPaint allow me to do that without me looking at it ? 

Thanks for your help.

**Comment by 61yyyyyy** [comment]
你好，请问你问题解决了吗？ 我在windows中遇到类似的问题，请问一下批处理功能中 ，三个图片地址应该存放在哪里才能被读取到？

**Comment by bak00700** [comment]
使用 CLI 进行批处理
您可以使用该命令批量处理镜像。执行 查看所有帮助信息。iopaint runiopaint run --help

--image：图像文件夹或图像文件路径
--mask：遮罩文件夹或文件路径。如果它是一个目录，则该目录中的遮罩图像应与原始图像同名。如果它是一个文件，则所有图像都将使用此遮罩。遮罩将自动调整为与原始图像相同的大小。
--output：输出目录或文件路径
